nservicebus/operations/async-analyzer.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Async Analyzer
+summary: How to use the async analyzer to avoid missing awaits
+component: Core
+versions: '[6,]'
+---
+
+Asynchronous APIs bring great performance improvements. However, they introduce new risks for bugs due to missed `await` statements. NServiceBus comes with a built in Roslyn analyzer to detect those missing awaits.
+
+## Functionality
+
+The analyzer will point out missing `await` statements when using asynchronous methods on the following classes and interfaces:
+* `IMessageHandlerContext`
+* `IMessageSession`
+* `Saga`
+* `IEndpointInstance`
+* `Endpoint`
+
+## Limitations
+
+This analyzer does only inspect the APIs contained on the classes and interfaces as described in the previous chapter. Other asynchronous operations on NServiceBus APIs are not analyzed.
```

Comment:
Section would be more appropriate than chapter.

Do we have anything to say about your other, non NSB-touching code that's dropping tasks?